source/Octopus.Client.E2ETests/NuSpecDependenciesFixture.cs

```diff
@@ -0,0 +1,221 @@
+                "//ns:package/ns:metadata/ns:dependencies/ns:group[@targetFramework = '.NETStandard2.0']",
+                nameSpaceManager);
+            netStandard20DependencyGroup.Should().NotBeNull("Should have a .NETStandard2.0 dependency group");
+        }
+
+        [Test]
+        public void NuSpecFileShouldHave4NetStandard20Dependencies()
+        {
+            var net45Dependencies = nuSpecFile.SelectNodes(
+                "//ns:package/ns:metadata/ns:dependencies/ns:group[@targetFramework = '.NETStandard2.0']/ns:dependency",
+                nameSpaceManager);
+            net45Dependencies.Count.Should().Be(4, "There should be 4 dependencies listed for .NETStandard2.0");
+        }
+
+        [Test]
+        public void NuSpecFileShouldHaveNetStandard20DependencyOnMicrosoftCSharp()
+        {
+            var dependency = nuSpecFile.SelectSingleNode(
+                "//ns:package/ns:metadata/ns:dependencies/ns:group[@targetFramework = '.NETStandard2.0']/ns:dependency[@id='Microsoft.CSharp']",
+                nameSpaceManager);
+            dependency.Should()
+                .NotBeNull("There should be a Microsoft.CSharp dependency for .NETStandard2.0");
+        }
+
+        [Test]
+        public void NuSpecFileShouldHaveNetStandard20DependencyOnSystemComponentModelAnnotations()
+        {
+            var dependency = nuSpecFile.SelectSingleNode(
+                "//ns:package/ns:metadata/ns:dependencies/ns:group[@targetFramework = '.NETStandard2.0']/ns:dependency[@id='System.ComponentModel.Annotations']",
+                nameSpaceManager);
+            dependency.Should()
+                .NotBeNull("There should be a System.ComponentModel.Annotations dependency for .NETStandard2.0");
+        }
+        
+        [Test]
+        public void NuSpecFileShouldHaveNetStandard20DependencyOnNewtonsoftJson()
+        {
+            var dependency = nuSpecFile.SelectSingleNode(
+                "//ns:package/ns:metadata/ns:dependencies/ns:group[@targetFramework = '.NETStandard2.0']/ns:dependency[@id='Newtonsoft.Json']",
+                nameSpaceManager);
+            dependency.Should()
+                .NotBeNull("There should be a Newtonsoft.Json dependency for .NETStandard2.0, as we cant inline it until we get to netcore 3");
+        }
+        
+        [Test]
+        public void NuSpecFileShouldHaveNetStandard20DependencyOnOctodiff()
+        {
+            var dependency = nuSpecFile.SelectSingleNode(
+                "//ns:package/ns:metadata/ns:dependencies/ns:group[@targetFramework = '.NETStandard2.0']/ns:dependency[@id='Octodiff']",
+                nameSpaceManager);
+            dependency.Should()
+                .NotBeNull("There should be a Octodiff dependency for .NETStandard2.0, as we cant inline it until we get to netcore 3");
+        }
+
+        [Test]
+        public void NetStandard20DependencyOnNewtonsoftJsonShouldBeSameVersionAsCsProj()
```

Comment:
This is why `dotnet pack` puts the csproj dependency versions in the nuspec for you. I bet this is going to catch us out if we have to update the Newtonsoft reference. It'll be up there with the "forgot approval files" commits :)